assets/templates/account/settings/settings_authorisation.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'account/settings/settings_base.html' %}
+
+{% block tab_authorisation %}active{% endblock %}
+
+
+{% block details %}
+
+    See password_change.html for code
+    This is in a seperate file for allauth
```

Comment:
Why?

Reply:
Allauth has views which you implemented, I didn't want to break on that with all custom views, however, to maintain some structure in the settings pages I started grouping them in a seperate folder which meant there was a disjoint in what to place where. This made, imo, the clearest solution.
"Why not account/settings/settings_base.html???"
As stated, I wanted all settings display logic in the settings folder, this includes the bit of code that autoselects the right tab which is in 'account/settings/settings_authorisation.html'

Reply:
All templates in the `account` folder are ours and should not be considered as part of allauth, although I get that that's probably not what you meant. This extra separation is very verbose in this case because they are already very close and therefore that does not weigh up to the extra files that are needed to make it work. The extra files make that more work is needed to understand what's going on (more files to follow).

Reply:
It is indeed not what I meant. What I meant was that the allauth views reference those files and as we did not customise those views, the exact location of those files can not be changed either.

Reply:
You can still just directly extend the base template, the extra indirection does not really help for clearability or maintainability.